src/jit/morph.cpp

```diff
@@ -2618,90 +2637,84 @@ GenTreeCall* Compiler::fgMorphArgs(GenTreeCall* callNode)
             numArgs++;
         }
 
-
         // insert nonstandard args (outside the calling convention)
 
 #if !defined(LEGACY_BACKEND) && !defined(_TARGET_X86_)
         // TODO-X86-CQ: Currently RyuJIT/x86 passes args on the stack, so this is not needed.
         // If/when we change that, the following code needs to be changed to correctly support the (TBD) managed calling
         // convention for x86/SSE.
-        if (!lateArgsComputed)
```

Comment:
If you look up the file a bit, you'll see that this is all within the `else` clause of a `if (lateArgsComputed)` case already.